subsys/bluetooth/audio/broadcast_sink.c

```diff
@@ -863,7 +858,11 @@ static void broadcast_sink_cleanup_streams(struct bt_audio_broadcast_sink *sink)
 		stream->codec = NULL;
 		stream->iso = NULL;
 		stream->group = NULL;
+
+		sys_slist_remove(&sink->streams, NULL, &stream->node);
```

Comment:
Additionally, we should change the `bt_audio_stream` member `node` node to be prefixed with `_` -> `_node`. So that it would not be used by the user to put it on some other list :D

Reply:
Agreed. I don't think we have any general guidelines for this in Zephyr (none that I am aware of), but the `_` does typically mean "internal". 

We do have a lot (a lot!) of fields in public structs that are internal and do not use the `_` prefix. 

We can easily fix that for ISO and LE Audio still, as those are experimental. Ideally we should be able to do this for all internal fields in public struct, as those should not be used by the applications, but I guess for the stable APIs we should still go through the proper process of changing those (if using `_` as a prefix is a convention we will start to follow

Reply:
Added the "_" prefix for all nodes in LE Audio (not that many yet).